test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1

```diff
@@ -55,19 +55,19 @@
         }
 
         It "non filesystem path using <parameter>" -TestCases @(
-            @{parameter="literalPath"},
-            @{parameter="path"}
+            @{parameter="literalPath"; expectedError='ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand'},
+            @{parameter="path"; expectedError='PathNotFound,Microsoft.PowerShell.Commands.SelectXmlCommand'}
         ) {
-            param($parameter)
-            $__data = "abcdefg"
-            $params = @{$parameter="variable:__data"}
-            { Select-XML @params "Root" | Should BeErrorId 'ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand' }
+            param($parameter, $expectedError)
+            $file = Join-Path -Path $testDrive -ChildPath 'abcdefg'
```

Comment:
this is changing the test, it's supposed to be a non-filesystem path hence it was using the variable provider

Reply:
I'll look at rewriting it. As written, the test does not actually accomplish that, variable:__data is being resolved to a path in the testdrive although I'm not sure how, yet.

Reply:
Fixed